src/components/Sections/Ourvoice/Ourvoice.js

```diff
@@ -0,0 +1,15 @@
+import Container from './Container';
```

Comment:
Please remember the translation, all the titles need to be in the translation file